lib/dotiw/methods.rb

```diff
@@ -38,12 +42,15 @@ def time_ago_in_words(from_time, include_seconds_or_options = {})
     private
 
     def options_with_scope(options)
-      options.merge!({:scope => DOTIW::DEFAULT_I18N_SCOPE_COMPACT}) if options.delete(:compact)
-      options
+      if options.key?(:compact)
+        options.merge({:scope => DOTIW::DEFAULT_I18N_SCOPE_COMPACT})
```

Comment:
`scope: DOTIW::DEFAULT_I18N_SCOPE_COMPACT`, let's fix this old syntax too at some point